<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>%{AUTHOR}</author>
 <class>VeracryptItemUi</class>
 <widget class="QWidget" name="VeracryptItemUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="horizontalGroupBox">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="nameLineEdit">
             <property name="placeholderText">
              <string>Change Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteConfigButton">
             <property name="text">
              <string>Delete Config</string>
             </property>
             <property name="icon">
              <iconset theme="delete">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="nameExistsInfoLabel">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;The name already exists !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fileRadioButton">
           <property name="text">
            <string>File</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="deviceRadioButton">
           <property name="text">
            <string>Device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filePushButton">
           <property name="text">
            <string>Select File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="devicePushButton">
           <property name="text">
            <string>Select Device</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="mountPathLabel">
             <property name="text">
              <string>Mount Path: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mountPath">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Select Location</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="keyFiles">
           <property name="title">
            <string>Key Files</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QListWidget" name="keyFileListWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="addKeyFileButton">
                <property name="text">
                 <string>Add Key File</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removeKeyFileButton">
                <property name="text">
                 <string>Remove Key File</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="passIntegrationLabel">
           <property name="text">
            <string>Path For Pass Integration (optional):</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="passIntegration"/>
           </item>
           <item>
            <widget class="QPushButton" name="passIntegrationSelectButton">
             <property name="text">
              <string>Select Path</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="moveUp">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="arrow-up">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QPushButton" name="moveDown">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="arrow-down">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
